### PR TITLE
[TACHYON-719]: Unmatched web resource path on LocalTachyonMaster

### PR DESCRIPTION
**Pull request by pfxuan**

### CODE REVIEW

**Comment by calvinjia** [comment]
@pfxuan Thanks for the fix, I'm curious does this actually slow down the unit tests since the UI gets launched?

**Comment by pfxuan** [comment]
@calvinjia It's a good suggestion. I didn't see any obvious overhead so far. But we could actually disable the launching of web server on LocalTachyonMaster.

**Comment by calvinjia** [comment]
@pfxuan that would be a cool feature to have, like a configuration option to enable the web ui. For this PR I think this is fine.

+1 LGTM

File: minicluster/src/main/java/tachyon/master/LocalTachyonMaster.java [hunk 1]
```diff
@@ -104,7 +104,8 @@ private LocalTachyonMaster(final String tachyonHome, TachyonConf tachyonConf)
     tachyonConf.set(Constants.HOST_RESOLUTION_TIMEOUT_MS, "250");
 
     tachyonConf.set(Constants.WEB_THREAD_COUNT, "1");
-    tachyonConf.set(Constants.WEB_RESOURCES, System.getProperty("user.dir") + "/src/main/webapp");
+    tachyonConf.set(Constants.WEB_RESOURCES, System.getProperty("user.dir")
+        + "/../servers/src/main/webapp");
```
Comment (apc999) [inline minicluster/src/main/java/tachyon/master/LocalTachyonMaster.java]: Actually, to have better portability, could we use `tachyon.util.io.PathUtils#concatPath` for this path calculation?

**Comment by apc999** [comment]
One minor comment left. Thanks for the PR!

File: minicluster/src/main/java/tachyon/master/LocalTachyonMaster.java (hunk 1 above)
Comment (pfxuan) [inline minicluster/src/main/java/tachyon/master/LocalTachyonMaster.java]: @apc999  Good suggestion! Thanks!

**Comment by pfxuan** [comment]
Jenkins, test this please.

**Comment by apc999** [comment]
+1 LGTM

**Pull request merged**